src/ui/public/pager/pager_service.factory.js

```diff
@@ -0,0 +1,12 @@
+import uiModules from 'ui/modules';
+import { PagerService } from './pager_service';
+
+const app = uiModules.get('kibana');
+
+app.factory('pagerService', () => {
```

Comment:
I think we should name this `pagerFactory` instead, which is a much clearer indication IMO that this object follows the factory pattern (i.e. it exposes `create` methods).